shared/test/otSdkDriver_test.js

```diff
@@ -159,6 +161,58 @@ describe("loop.OTSdkDriver", function() {
     });
   });
 
+  describe("#initPublisherComplete", function() {
+    beforeEach(function() {
+      sandbox.stub(publisher, "off");
+      driver.setupStreamElements(new sharedActions.SetupStreamElements({
+        publisherConfig: publisherConfig
+      }));
+    });
+
+    it("should call initPublisher", function() {
+      sinon.assert.calledOnce(sdk.initPublisher);
```

Comment:
This will be able to be dropped once you've merged the two, as its already tested in the `setupStreamElements` describe.